addons/fleet/models/fleet_vehicle.py

```diff
@@ -96,8 +96,6 @@ def _get_default_state(self):
         string='Has Contracts to renew')
     contract_renewal_overdue = fields.Boolean(compute='_compute_contract_reminder', search='_search_get_overdue_contract_reminder',
         string='Has Contracts Overdue')
-    contract_renewal_name = fields.Text(compute='_compute_contract_reminder', string='Name of contract to renew soon')
```

Comment:
As you remove this field here, you can remove it wherever it occurs ctrl-shift-f on the repository :)